resources/src/main/res/layout/git_dialog_layout.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout      
+	xmlns:android="http://schemas.android.com/apk/res/android"
+	android:gravity="center"
+	android:orientation="vertical"
+	android:padding="10.0dip"    
+	android:layout_width="fill_parent"
+	android:layout_height="fill_parent">
+
+	<com.google.android.material.textfield.TextInputLayout
+		android:id="@+id/textInputRepo"
+		style="@style/Widget.Material3.TextInputEditText.OutlinedBox.Dense"
+		android:theme="@style/AppTheme.TextInputLayout"
+		android:layout_width="fill_parent"
+		android:layout_height="wrap_content"
+		android:hint="@string/repository_url">
+
+		<com.google.android.material.textfield.TextInputEditText
+			android:id="@+id/textRepo"
+			android:layout_width="fill_parent"
+			android:layout_height="wrap_content"
+			android:ems="10"/>
+
+	</com.google.android.material.textfield.TextInputLayout>
+
+</LinearLayout>
```

Comment:
This whole layout file is redundant. You could replace usage of this file with `layout_dialog_text_input`.